### ISSUE TITLE
Bad PID 

### ISSUE THREAD

**Issue by TipanLogic** [arkmanager/ark-server-tools#1239]
Getting error when trying to start server
steam@server-main:~$ arkmanager run
Running command 'run' for instance 'main'
2022-12-20 01:16:39: start
2022-12-20 01:16:39: Running /media/user/Файлопом/user/ARK/ShooterGame/Binaries/Linux/ShooterGameServer TheIsland\?RCONEnabled=True\?RCONPort=32330\?SessionName=ARK\ Server\ Tools\?Port=7778\?QueryPort=27015\?ServerPassword\?ServerAdminPassword=keyboardcat\?MaxPlayers=70\?ServerPassword\?listen
2022-12-20 01:16:39: Server PID: 15257
Shutdown handler: initalize.
Using binned.
4.5.1-0+UE4 7038 3077 404 10
/usr/local/bin/arkmanager: line 1314: 15257 Emergency stop         "$arkserverroot/$arkserverexec" "$arkserveropts" "${arkextraopts[@]}"
2022-12-20 01:16:44: Bad PID ''; expected '15257'
2022-12-20 01:16:44: exited with status 0

**Comment by klightspeed** [comment]
`SIGABRT` (`Aborted` / `Аварийный останов` (https://github.com/bminor/glibc/blob/master/po/ru.po#L5423-L5425) -> `Emergency stop`) is just the signal that the game server uses to terminate itself, and the `Bad PID ''` is the script indicating that it no longer finds the server running.

Unfortunately there's not much to go on to say why the game server is exiting after 5 seconds.

**Comment by TipanLogic** [comment]
We are using Debian 11 
1st try to run server was with error with message that something was wrong with '.uproject' file, we solved it by reinstallation of steamcmd with user 'steam'
2nd try was with error message 'No protocols specified', we solved it by running server under user 'steam'
And that's the 3rd error we trying to solve...

**Comment by klightspeed** [comment]
It sounds like there is still potentially something wrong with the ARK server install.  Please try running `arkmanager update --validate` to validate the install files.  If that still doesn't work, please try setting `LC_ALL=C` when running arkmanager to eliminate collation / character types being an issue.

**Comment by TipanLogic** [comment]
Could it be that the error is caused by the presence of Cyrillic symbols in path?

**Comment by klightspeed** [comment]
Yes, it is quite possible that the ARK / Unreal Engine 4 server does not properly handle paths with characters that have the high bit set (as is the case with UTF-8, KOI8, or ISO-8859-5 encoded Cyrillic characters).  We know it doesn't handle characters with the high bit set on the command-line or in its config file (where it only understands either 7-bit ASCII or 16-bit UTF-16).
